### DISCUSSION TITLE
Firefox Nightly - 2013-2017 Logo ( Big Sur Icon )

### DISCUSSION THREAD

**Discussion by sulflix** [elrumo/macOS_Big_Sur_icons_replacements#1491] [Icon request]
https://logos.fandom.com/wiki/Firefox_Nightly#2013%E2%80%932017
![image](https://github.com/elrumo/macOS_Big_Sur_icons_replacements/assets/84033342/5cf8ea18-23ac-4006-8dbc-6382de6985d9)

**Comment by vikasbhatia** [comment]
Here you go, two options:
<img width="128" alt="Firefox Nightly 1" src="https://github.com/user-attachments/assets/d087c159-fc21-498f-86fe-5645c7e35895">
<img width="128" alt="Firefox Nightly 2" src="https://github.com/user-attachments/assets/f8ffd928-6240-4ba6-8037-c945936fff4b">

You can download the PNGs here to start using already, and the ICNS files from [my profile](https://macosicons.com/#/u/sidewinder) later when they are approved.
